docs/installation/installation-detailed.rst

```diff
@@ -22,8 +22,7 @@ Detailed installation guide
 
 .. admonition:: CAVEAT LECTOR
 
-   Invenio v3.0 alpha is a bleeding-edge developer preview version that is
-   scheduled for public release in Q3/2016.
+   Invenio v3.0 alpha is a bleeding-edge developer preview version.
```

Comment:
I have removed the deadline from all places since we have missed all passed deadlines so far. Honestly, I don't think it is good idea to give ETA if we don't have clearly defined roadmap with allocated resources.

--
TODO list: https://github.com/issues?utf8=%E2%9C%93&q=org%3Ainveniosoftware+is%3Aopen+milestone%3AI3B-API-Docs